sites/svelte.dev/src/routes/content.json/content.server.js

```diff
@@ -48,6 +48,14 @@ export async function content() {
 		const intro = sections?.shift()?.trim();
 		const rank = +metadata.rank;
 
+		// Push the main page into it
```

Comment:
this looks like exactly what is in the `if` block that comes right after this. do we need both? it looks like this will only have when `!intro` in which case won't `await plaintext(intro)` be empty or something?

Reply:
Fixed. I removed `if(intro)` condition, that fixes it. However we should add intros to 7 pages, which don't, just for documentation side of things